sql/schema_changer.go

```diff
@@ -148,16 +148,16 @@ func (sc *SchemaChanger) findTableWithLease(txn *client.Txn, lease TableDescript
 
 // ReleaseLease the table lease if it is the one registered with
 // the table descriptor.
-func (sc *SchemaChanger) ReleaseLease(lease TableDescriptor_SchemaChangeLease) *roachpb.Error {
+func (sc *SchemaChanger) ReleaseLease(lease TableDescriptor_SchemaChangeLease) error {
 	return sc.db.Txn(func(txn *client.Txn) *roachpb.Error {
-		tableDesc, err := sc.findTableWithLease(txn, lease)
-		if err != nil {
-			return err
+		tableDesc, pErr := sc.findTableWithLease(txn, lease)
+		if pErr != nil {
+			return pErr
 		}
 		tableDesc.Lease = nil
 		txn.SetSystemConfigTrigger()
 		return txn.Put(MakeDescMetadataKey(tableDesc.ID), wrapDescriptor(tableDesc))
-	})
+	}).GoError()
```

Comment:
I like this more than assigning to `pErr` first and then calling `GoError()`. There are some other occurrences.